app/routers/users.py

```
@@ -0,0 +1,64 @@
+
+@router.post("/", summary="유저 회원가입",description="회원가입: 이메일, 비밀번호, 닉네임, 프로필 이미지(선택)로 가입", tags=["users"])
+async def post_user(new_user: UserSignupRequest):
+    users = read_users()
+
+    new_user = {
+        "id": len(users) + 1,
+        "email": new_user.email,
+        "nickname": new_user.nickname,
+        "password_hash": hash_password(new_user.password),
+        "profile_image": new_user.profile_image,
+        "created_at": datetime.utcnow().isoformat() + "Z"
```

Ruff (0.14.11)

`datetime.datetime.utcnow()` used

(DTZ003)